### PR TITLE
team: add invite user page

### PR DESCRIPTION
**Pull request by dagoof**

### CODE REVIEW

File: assets/src/console/pages/services/TeamInvitePage.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        <>
+          <Field name="email" label="Email address">
+            {({ input, ariaProps, ...rest }) => (
+              <BasicTextInput {...input} {...ariaProps} {...rest} />
+            )}
+          </Field>
+
+          <Fieldset legend="Permission level">
+            {roles.map(role => (
+              <Radio
```
Comment (jonathaningram) [inline assets/src/console/pages/services/TeamInvitePage.tsx]: react `key` prop?

Comment (jonathaningram) [reply assets/src/console/pages/services/TeamInvitePage.tsx]: oops can't read.

File: assets/src/console/pages/services/TeamInvitePage.tsx [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+      {inviteError ? (
+        <ErrorAlert
+          title="Error inviting user"
+          message={inviteError && inviteError.message}
+          showError
+        ></ErrorAlert>
+      ) : null}
+
+      <Form<FormValues>
```
Comment (jonathaningram) [inline assets/src/console/pages/services/TeamInvitePage.tsx]: did you want any form validation on this form yet, or later?

Comment (dagoof) [reply assets/src/console/pages/services/TeamInvitePage.tsx]: will do later

**Pull request merged**